apps/meteor/client/views/admin/moderation/UserReports/UserReportInfo.tsx

```diff
@@ -0,0 +1,158 @@
+import type { IUser, UserReport } from '@rocket.chat/core-typings';
+import {
+	Box,
+	Callout,
+	StatesAction,
+	StatesActions,
+	StatesIcon,
+	StatesTitle,
+	ContextualbarFooter,
+	FieldGroup,
+	Field,
+	FieldLabel,
+	FieldRow,
+	ContextualbarSkeleton,
+} from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import { useEndpoint, useToastMessageDispatch, useTranslation } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+import React, { useMemo } from 'react';
+
+import GenericNoResults from '../../../../components/GenericNoResults';
+import UserCard from '../../../../components/UserCard';
+import { useFormatDate } from '../../../../hooks/useFormatDate';
+import ReportReason from '../helpers/ReportReason';
+import UserColumn from '../helpers/UserColumn';
+import UserContextFooter from './UserContextFooter';
+
+const isUser = (obj: any): obj is IUser => {
+	if (obj?.createdAt && typeof obj.createdAt === 'string') {
+		obj.createdAt = new Date(obj.createdAt);
+	}
+	return obj?.createdAt instanceof Date;
+};
+
+const UserReportInfo = ({ userId }: { userId: string }): JSX.Element => {
+	const t = useTranslation();
+	const dispatchToastMessage = useToastMessageDispatch();
+	const getUserReports = useEndpoint('GET', '/v1/moderation.user.reportsByUserId');
+	const formatDateAndTime = useFormatDate();
+
+	const {
+		data: report,
+		refetch: reloadUsersReports,
+		isLoading: isLoadingUsersReports,
+		isSuccess: isSuccessUsersReports,
+		isError,
+		dataUpdatedAt,
+	} = useQuery(
+		['moderation.usersReport', { userId }],
+		async () => {
+			const reports = await getUserReports({ userId });
+			return reports;
+		},
+		{
+			onError: (error) => {
```

Comment:
No need to this since you're rendering the error on the UI

Reply:
Thank you, done!